rocAL/rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -25,34 +25,28 @@ THE SOFTWARE.
 #include "rocal_api_types.h"
 
 /*!
- * \brief Rearranges the order of the frames in the sequences with respect to new_order.
- * Frames can be repeated or dropped in the new_order.
+ * \brief Rearranges the order of the frames in the sequences with respect to new_order. new_order can have values in the range [0, sequence_length). Frames can be repeated or dropped in the new_order.
  * \ingroup group_rocal_augmentations
- * \note Accepts U8 and RGB24 input.
- * \param [in] p_context context for the pipeline.
- * \param [in] p_input Input Rocal Tensor
- * \param [in] new_order represents the new order of the frames in the sequence in the range [0, sequence_length)
- * \param [in] is_output True: the output tensor is needed by user and will be copied to output buffers using the data
- * transfer API calls. False: the output tensor is just an intermediate tensor, user is not interested in
- * using it directly. This option allows certain optimizations to be achieved.
- * \return RocalTensor
+ * \note: Accepts U8 and RGB24 input.
+ * \param [in] context context for the pipeline.
+ * \param [in] input Input Rocal Tensor
+ * \param [in] new_order represents the new order of the frames in the sequence
+ * \param [in] is_output True: the output image is needed by user and will be copied to output buffers using the data transfer API calls. False: the output image is just an intermediate image, user is not interested in using it directly. This option allows certain optimizations to be achieved.
+ * \return
  */
 extern "C" RocalTensor ROCAL_API_CALL rocalSequenceRearrange(RocalContext p_context, RocalTensor p_input,
-                                                              std::vector<unsigned int>& new_order,
-                                                              bool is_output);
+                                                             std::vector<unsigned int> &new_order,
+                                                             bool is_output);
 
-/*! \brief Resize images.
+/*! \brief Accepts U8 and RGB24 input.
```

Comment:
I think this briefing should be retained as Resize images.
And this to be added in \note as above augmentation